Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -888,6 +914,17 @@ class TabCell: UICollectionViewCell, TabTrayCell {
             make.left.right.equalTo(backgroundHolder)
             make.bottom.equalTo(backgroundHolder.snp.bottom)
         }
+        
+        faviconBG.snp.makeConstraints { make in
+            make.centerY.equalToSuperview().offset(10)
+            make.centerX.equalToSuperview()
+            make.size.equalTo(TopSiteCellUX.BackgroundSize)
+        }
+        
+        smallFaviconView.snp.makeConstraints { make in
+            make.size.equalTo(TopSiteCellUX.IconSize)
+            make.center.equalTo(faviconBG)
+        }
```

Comment:
Let's not user SnapKit, especially because we're in the active process of removing it.

Or is it difficult to set layout constraints and we'd have to completely rewrite the constraints in this file to achieve that?

I'm personally for the rewrite route, but, if that's not something we have time right now, I'm happy to let it go and we can re-write when we remove SnapKit from this file.

Reply:
Snapkit from TabCell will be removed in a follow-up refactor PR related to tab groups where this class is also used and needs to be moved.